practice/practice-02/src/styles/layouts/_header.scss

```diff
@@ -0,0 +1,27 @@
+.header {
+  position: relative;
+  display: flex;
```

Comment:
I saw you use alot of flex style, could you split it for common uses?

Reply:
Fixed.